db/channel_cache_single_test.go

```diff
@@ -240,7 +276,7 @@ func TestPrependChanges(t *testing.T) {
 	require.Len(t, cachedChanges, 3)
 
 	// 2. Test prepend to populated cache, with overlap and duplicates
-	cache = newSingleChannelCache(dbCtx, "PrependPopulatedCache", 0, (base.NewSyncGatewayStats()).NewDBStats("", false, false, false).Cache())
+	cache = newSingleChannelCache(dbCtx, "PrependPopulatedCache", 0, dbstats.Cache())
```

Comment:
I think this (and the below) are fine... Just note that I think the behaviour is different, previously we were creating new stats, now re-using. If test passes tho its fine.